src/core/stats.cc

```diff
@@ -1336,8 +1340,18 @@ py::oobj Stats::get_stat_as_pyobject(Stat stat) {
     case Stat::NModal: {
       return pywrap_stat<size_t>(stat);
     }
+    case Stat::Mean: {
+      switch (dt::stype_to_ltype(column->stype())) {
```

Comment:
This should really be `if (column->type()->is_temporal()) { ... }`,

where `Type::is_temporal()` is renamed from `Type::is_time()` (the current name is too confusing).

Reply:
Yeah, I've been refactoring this `switch` and forgot to replace it with a simple `if` at the end.

Reply:
Btw, `TypeImpl::is_time()` should be also replaced with `TypeImpl:: is_temporal()`.